Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -59,11 +59,52 @@ public protocol HTTPNetworkTransportRetryDelegate: HTTPNetworkTransportDelegate
   ///   - request: The URLRequest which generated the error
   ///   - response: [Optional] Any response received when the error was generated
   ///   - retryHandler: A closure indicating whether the operation should be retried. Asyncrhonous to allow for re-authentication or other async operations to complete.
+  @available(*, deprecated, message: "Use networkTransport(_:receivedError:for:,response:continueHandler:) instead")
```

Comment:
So this is going to be a bit of a mess because even if we deprecate this method, people are still going to have the deprecation warning on the default implementation below. 

I'm wondering if this might be a "Let's take advantage of the fact that we're not at 1.0" moment to just straight up replace this method.

Reply:
hmm, I don't see deprecation warning unless I implement method in my delegate. Or I'm missing something?

Reply:
Hm...in theory this should still cause a deprecation warning where it's in the same file but maybe that's been disabled to prevent exactly this scenario...